### PR TITLE
Added shutdown handler for Classic Server

### PR DESCRIPTION
**Pull request by Zhdanov0**
Added shutdown handler for server to avoid active connections after termination.
If you start the server with classic mode, then at the end of the server, there are still active connections.

### CODE REVIEW

File: src/remote/server/os/posix/inet_server.cpp [hunk 1]
```diff
@@ -314,6 +315,10 @@ int CLIB_ROUTINE main( int argc, char** argv)
 		// activate paths set with -e family of switches
 		ISC_set_prefix(0, 0);
 
+		// set shutdown signals handler for listener
```
Comment (AlexPeshkoff) [inline src/remote/server/os/posix/inet_server.cpp:318]: Does it make sense to add this handlers in super & superclassic modes?

Comment (Zhdanov0) [reply src/remote/server/os/posix/inet_server.cpp:318]: No, it's only relevant to classic

Comment (Zhdanov0) [reply src/remote/server/os/posix/inet_server.cpp:318]: I'll add a condition

**Review by AlexPeshkoff — APPROVED** [review]

**Comment by AlexPeshkoff** [comment]
I suggest to backport it to 5 & 4.

**Comment by dyemanov** [comment]
Backported.

**Pull request merged**